script/test

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+
+trap cleanup 1 2 15
+
+cleanup() {
+  rm -rf "$temp_folder"
+}
+
+temp_folder="./test/tmp"
+fixtures_folder="./test/scss/fixtures"
```

Comment:
You should probably define those above `cleanup`. And I guess it's best practice to write them in all caps.